internal/repos/modules.go

```diff
@@ -76,7 +112,11 @@ func importRepoRulesModules(filename string) (repos []Repo, err error) {
 			continue
 		}
 
-		repos = append(repos, toRepoRule(mod))
+		repo, err := toRepoRule(mod)
```

Comment:
Since this is going to be hitting the network now, these conversions should happen in parallel.